CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.24)
+cmake_minimum_required(VERSION 3.23)
```

Comment:
Clion built-in cmake has this version, so if you don't care, I prefer that. Otherwise happy to bump to what's convenient for you and fix my cmake setup.

Reply:
I suspect my CLion's CMake was a bit newer when I created the file. We are not using any fancy CMake features though, we can probably also drop it down much lower.